naga/src/back/spv/index.rs

```diff
@@ -38,98 +38,160 @@ impl<'w> BlockContext<'w> {
     ///
     /// Given `array`, an expression referring a runtime-sized array, return the
     /// instruction id for the array's length.
+    ///
+    /// Runtime-sized arrays may only appear in the values of global
+    /// variables, which must have one of the following Naga types:
+    /// 1) A runtime-sized array.
+    /// 2) A struct whose last member is a runtime-sized array.
+    /// 3) A binding array of 2).
```

Comment:
thought: I was greatly surprised to learn that the `1)` form is accepted by Rustdoc as valid ordered list syntax!

style(non-blocking): I think it'd be better to use the `1.` syntax form, as that's more ubiquitous, but I notice that _most_ (not all 🤔) of Naga uses the `1)` form.

style(non-blocking): I find it nicer to have a newline between a list and other surrounding block content. This matches better with the styling presented in most tools I'm aware of, incl. Rustdoc.